tooling.yml

```diff
@@ -0,0 +1,5 @@
+architecture-sniffer:
+    priority: 2
+
+code-sniffer:
+    level: 1
```

Comment:
* Raise to 2nd.

Reply:
I can't because of the bridge to `UtilEncodingService` module. Sniffer always complains about the bridge methods definitions.

Reply:
Resolved.